packages/commercetools-sdk-middleware-http/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Any reason why the file names have `commercetools` in them?

Shouldn't that be more explicit